www/js/components/pages/vm/fwrules-list.js

```diff
@@ -74,18 +119,18 @@ var FWRulesList = require('../../../views/collection').extend({
      * @param  {Object} options.vm VM object to scope fw rules
      * @param  {string} options.user User UUID to scope fw rules
      */
-    initialize: function(options) {
+    initialize: function (options) {
         var app = this.app = options.app;
         if (options.vm) {
-            this.collection = new FWRules(null, {params: { vm_uuid: options.vm.get('uuid') }});
+            this.collection = new FWRules(null, {params: {vm_uuid: options.vm.get('uuid')}});
         } else if (options.user) {
-            this.collection = new FWRules(null, {params: { owner_uuid: options.user }});
+            this.collection = new FWRules(null, {params: {owner_uuid: options.user}});
         } else {
             this.collection = new FWRules();
         }
 
-        this.on('itemview:disable:rule', function(iv) {
-            iv.model.on('sync', function() {
+        this.on('itemview:disable:rule', function (iv) {
+            iv.model.on('sync', function () {
                 app.vent.trigger('notification', {
```

Comment:
DRY 
`app.vent.trigger('notification', {
                      level: 'success',		                      level: 'success',
                      message: "Firewall rule enabled successfully."		                      message: "Firewall rule enabled successfully."
                 });		                 });
                 this.collection.fetch({reset: true});		                 this.collection.fetch({reset: true});`
to function